org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/nexus/NexusScanFileManager.java

```diff
@@ -210,10 +210,17 @@ public NexusScanInfo getNexusScanInfo() {
 	}
 	
 	@Override
-	public void positionPerformed(PositionEvent evt) throws ScanningException {
-		solsticeScanMonitor.setPosition(null, evt.getPosition());
+	public void positionPerformed(PositionEvent event) throws ScanningException {
+		solsticeScanMonitor.setPosition(null, event.getPosition());
 	}
-
+	
+	@Override
+	public void positionMovePerformed(PositionEvent event) throws ScanningException {
+		if (solsticeScanMonitor.writeAfterMovePerformed()) {
+			solsticeScanMonitor.setPosition(null, event.getPosition());
+		}
+	}
+	
 	protected SolsticeScanMonitor createSolsticeScanMonitor(ScanModel model) {
 		SolsticeScanMonitor solsticeScanMonitor = new SolsticeScanMonitor(model);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "solsticeScanMonitor" which hides the field declared at line 82. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)